pycqed/tests/analysis_v2/test_residual_ZZ_analysis.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import numpy as np
+import pycqed as pq
+import os
+from pycqed.analysis import measurement_analysis as ma
+from pycqed.analysis_v2 import residual_ZZ_analysis as rza
+
+class Test_residual_ZZ_analysis(unittest.TestCase):
+
+	@classmethod
+	def setUpClass(self):
+		self.datadir = os.path.join(pq.__path__[0], 'tests', 'test_data')
+		ma.a_tools.datadir = self.datadir
+
+	def test_matrix_diagonal(self):
+		# Test if diagonal matrix elements are nan
+		qubit_labels = ['D1', 'Z1', 'X', 'D3']
+		ts_list = ma.a_tools.get_timestamps_in_range('20200124_182258','20200124_183814', '')
+		a = rza.Residual_Crosstalk(qubits=qubit_labels, t_start=ts_list[0], t_stop=ts_list[-1])
+		for i in range(len(qubit_labels)):
+			self.assertTrue(np.isnan(a.proc_data_dict['quantities_of_interest']['matrix'][i,i]))
+
+	def test_matrix_non_diagonal(self):
+		# Test if non-diagonal matrix elements are close to expected data values
+		qubit_labels = ['D1', 'Z1', 'X', 'D3']
+		ts_list = ma.a_tools.get_timestamps_in_range('20200124_182258','20200124_183814', '')
+		a = rza.Residual_Crosstalk(qubits=qubit_labels, t_start=ts_list[0], t_stop=ts_list[-1])
```

Comment:
OK, so here I see that one needs a range of timestamps. 
What are the conditions these timestamps need to have in order to be able to run the analysis? These conditions for when the analysis works are not documented anywhere.